pulumi/grapl/__main__.py

```diff
@@ -134,9 +134,22 @@ def main() -> None:
     nomad_provider: pulumi.ProviderResource | None = None
     consul_provider: pulumi.ProviderResource | None = None
 
+    # Use the nomad ALB address or default to the local host address.
+    # Apparently nomad templates use a different templating syntax than anywhere else
+    nomad_endpoint: pulumi.Output[
+        str
+    ] | str = '{{ env "attr.unique.network.ip-address" }}:4646'
+
     if not config.LOCAL_GRAPL:
         upstream_stacks = UpstreamStacks()
         nomad_provider = get_nomad_provider_address(upstream_stacks.nomad_server)
+
+        nomad_endpoint = nomad_provider.address
+        # we need to strip off the prepended http:// because that combined with a non-standard port is
+        nomad_endpoint = pulumi.Output.all(
+            nomad_endpoint=nomad_endpoint,
+        ).apply(lambda endpoint: f"{nomad_endpoint}".replace("http://", ""))
```

Comment:
Is there any possibility of this being `https://` at any point in the future?

I think there are URL libraries that can be used to strip the protocol portion off a URL generically... might be worthwhile here.

Reply:
I see what you're saying. I don't think it makes sense to pull in a dependency just to do this. Let me test using .split on '://'

Reply:
I think you can just use [urllib.parse](https://docs.python.org/3/library/urllib.parse.html#urllib.parse.urlparse):
```python
urllib.parse(nomad_endpoint).netloc
```